src/treeview/nodes/tests.ts

```diff
@@ -21,39 +26,58 @@ export class TestRootNode extends BaseNode {
     item.collapsibleState =
       this.tests.length === 0 ? vscode.TreeItemCollapsibleState.None : vscode.TreeItemCollapsibleState.Collapsed;
 
+    // To key in to "when": "view == meson-project && viewItem == meson-test-root" in package.json.
+    item.contextValue = "meson-test-root";
+
     return item;
   }
 
   override getChildren() {
     return this.tests.map((test) => new TestNode(this.id, test, this.isBenchmark));
   }
+
+  run() {
```

Comment:
Can you add a return type?

Reply:
Can do, though it's from the `IRunnable` interface; not sure what the gain is other than more stuff to edit if the interface changes.

Reply:
To me, the gain is explicit return types. Helps when reading code. Everyone has different preferences though.